src/stories/HdTextarea.stories.js

```diff
@@ -13,6 +13,7 @@ storiesOf('Components/Form/HdTextarea', module)
       <HdTextarea
         v-model="value"
         :required="true"
+        :maxlength="500"
```

Comment:
```suggestion
```

It might seem like a default behavior, it's better if we keep it in its own story.

Reply:
oh right, i already created another case for that. you can see the changes here: https://github.com/homeday-de/homeday-blocks/pull/815/commits/504d02ceac39fc5ee2fdceb9ef2a76471a98a41c